src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -19,11 +19,24 @@ public CinemaHall add(CinemaHall cinemaHall) {
 
     @Override
     public CinemaHall get(Long id) {
-        return cinemaHallDao.get(id).get();
+        return cinemaHallDao.get(id).orElseThrow(
+                () -> new RuntimeException("cinemaHall not found"));
     }
 
     @Override
     public List<CinemaHall> getAll() {
         return cinemaHallDao.getAll();
     }
+
+    @Override
+    public Boolean update(CinemaHall cinemaHall) {
```

Comment:
Consider changing the return type of the `update` method to `void`. The current approach of returning a boolean by comparing entities is not reliable for determining if the update was successful.